generator/ServiceClientGeneratorLib/Member.cs

```diff
@@ -247,6 +247,15 @@ public string MarshallName
         {
             get
             {
+                // For Json11 and Json10 protocols the name of the member is always the MarshallName because these protocols don't support
```

Comment:
No need to change, but could this be simpler to avoid repeating `return LocationName ?? _defaultMarshallName` twice?

Like this: 
```csharp
if (string.Equals(this.model.Protocol, "json", StringComparison.Ordinal))
{
    // Only change the marshall name if no shape modifier exists
    if (this.model.Customizations.GetShapeModifier(this.OwningShape.Name) == null)
    {
        return this.ModeledName;
    }
}